src/main/java/juja/microservices/users/service/UserService.java

```diff
@@ -59,9 +59,15 @@ private UserDTO convertGetAllUsersDto(User user) {
         logger.debug("Received response from repository: {}", users.toString());
 
         logger.debug("Converting received user list to userDTO list");
-        List<UserDTO> result =users.stream()
-                .map(this::convertGetUuidBySlackDto)
-                .collect(Collectors.toList());
+        List<UserDTO> result;
+        try {
+            result = users.stream()
+                    .map(this::convertGetUuidBySlackDto)
+                    .collect(Collectors.toList());
+        } catch (NullPointerException e) {
+            logger.warn("Users" + request.getSlackNames() + "isn't found. Received empty list from repository.");
+            throw new UserException("User" + request.getSlackNames() + "isn't found by your request!");
```

Comment:
And I think you should use Past Simple Passive, instead Present Simple Passive :)
for example: Users weren't found.